dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/Enrollment.java

```diff
@@ -79,25 +78,19 @@
     private String orgUnit;
 
     @JsonProperty
-    private String orgUnitName;
+    private String enrolledAt;
```

Comment:
I have a feeling this is going to be a souce of confusion. This could sound more of a date than orgUnit as many of the date properties are now renamed created|updated|...|At.